shared/feature/feedconfirmation/src/main/java/com/epmedu/animeal/feedconfirmation/presentation/FeedConfirmationUI.kt

```diff
@@ -0,0 +1,87 @@
+package com.epmedu.animeal.feedconfirmation.presentation
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import com.epmedu.animeal.feedconfirmation.R
+import com.epmedu.animeal.foundation.button.AnimealButton
+import com.epmedu.animeal.foundation.button.AnimealSecondaryButton
+import com.epmedu.animeal.foundation.preview.AnimealPreview
+import com.epmedu.animeal.foundation.theme.AnimealTheme
+
+@Composable
+internal fun FeedConfirmationUI(
+    onAgreeClick: () -> Unit,
+    onCancelClick: () -> Unit
+) {
+    Scaffold(
+        modifier = Modifier
+            .statusBarsPadding()
```

Comment:
You must use status bar padding on the top element of the screen, not on the root element, otherwise there will be status bar blinking on navigation between screens

Reply:
Copy that. Should I put it on the Column element then?

Reply:
Yep